<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TravelSearchWindow</class>
 <widget class="QMainWindow" name="TravelSearchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Travel Search</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>426</width>
         <height>367</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>From:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="btnPlaceFrom">
              <property name="text">
               <string>Select Departure</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnFromFavorite">
              <property name="minimumSize">
               <size>
                <width>64</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>64</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="trafikanten.qrc">
                <normaloff>:/images/favorites.png</normaloff>:/images/favorites.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>48</width>
                <height>48</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>To:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="btnPlaceTo">
              <property name="text">
               <string>Select Destination</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnToFavorite">
              <property name="minimumSize">
               <size>
                <width>64</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>64</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="trafikanten.qrc">
                <normaloff>:/images/favorites.png</normaloff>:/images/favorites.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>48</width>
                <height>48</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="dateTimeVerticalContainer">
          <item>
           <layout class="QHBoxLayout" name="dateTimeHorizontalContainer">
            <item>
             <widget class="QPushButton" name="btnNow">
              <property name="text">
               <string>Now</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblNoResultsFounds">
          <property name="text">
           <string>No travels found.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tblResults">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>12</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>446</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAddFavorite"/>
    <addaction name="actionSwitch_direction"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionAddFavorite">
   <property name="text">
    <string>Add as favorite</string>
   </property>
  </action>
  <action name="actionSwitch_direction">
   <property name="text">
    <string>Switch direction</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="trafikanten.qrc"/>
 </resources>
 <connections/>
</ui>
